### DISCUSSION TITLE
Local building:  App Fingerprint

### DISCUSSION THREAD

**Discussion by wir3z** [owntracks/android#1536] [Q&A] [answered]
I've grabbed a Google Maps API key, and am trying to secure it to the app.  In the `CONTRIBUTING.md` file, it mentions that during startup, and exception should log the build fingerprint in the logs.  Is that in the device logs in the app?  I'm not seeing any reference to the fingerprint anywhere.

**Comment by growse** [comment, marked as answer]
You'll see it in the adb device logs.

**Reply by wir3z** [reply]
Awesome, I found it.  Thanks for the clarity!
